zkevm-circuits/src/ecc_circuit.rs

```diff
@@ -473,79 +621,77 @@ impl<F: Field, const XI_0: i64> EccCircuit<F, XI_0> {
         );
 
         // sum1 = if P == (0, 0) then r else r + P
-        let sum1 = ecc_chip.add_unequal(ctx, &rand_point, &point_p.decomposed.ec_point, true);
+        let sum1 = ecc_chip.add_unequal(ctx, &rand_point, &point_p.ec_point, true);
         let sum1 = ecc_chip.select(ctx, &rand_point, &sum1, &point_p_is_zero);
 
         // sum2 = if Q == (0, 0) then sum1 else sum1 + Q
-        let sum2 = ecc_chip.add_unequal(ctx, &sum1, &point_q.decomposed.ec_point, true);
+        let sum2 = ecc_chip.add_unequal(ctx, &sum1, &point_q.ec_point, true);
         let sum2 = ecc_chip.select(ctx, &sum1, &sum2, &point_q_is_zero);
 
         // sum3 = if R == (0, 0) then sum2 else sum2 - R
-        let sum3 = ecc_chip.sub_unequal(ctx, &sum2, &point_r.decomposed.ec_point, true);
+        let sum3 = ecc_chip.sub_unequal(ctx, &sum2, &point_r.ec_point, true);
         let sum3 = ecc_chip.select(ctx, &sum2, &sum3, &point_r_is_zero);
 
         ecc_chip.assert_equal(ctx, &rand_point, &sum3);
 
         log::trace!("[ECC] EcAdd Assignmnet END:");
         log_context_cursor!(ctx);
-        EcAddAssigned {
+        EcAddDecomposed {
             point_p,
             point_q,
             point_r,
         }
     }
 
     #[allow(clippy::too_many_arguments)]
-    fn assign_ec_mul_op(
+    fn decompose_ec_mul_op(
```

Comment:
can you please add some comments, for each functions, for the desired behavior for decompose/assign?